canonical-web-and-design/maas-ui
# feat(all): add 3.0 Usabilla campaign

## Done

- Update the feedback button for 3.0.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Open `shared/src/components/Footer/Footer.tsx`
- After line 26 add `debug = false;`
- Start the ui.
- Go to Settings.
- In General check that the analytics option is turned on.
- Scroll down to the footer and click "Give feedback".
- You should see the Usabilla feedback panel.

## Fixes

Fixes: canonical-web-and-design/maas-squad#2379.
